### DISCUSSION TITLE
Install system-wide

### DISCUSSION THREAD

**Discussion by ghecko** [ThePorgs/Exegol#188] [General]
Is it possible to install exegol system-wide? It could be interesting for working with exegol on a shared machine

**Comment by ghecko** [comment]
WIth ability to manage configuration system-wide (such as /etc/exegol/exegol.conf) so that every user share workspaces and custom resources

**Comment by Dramelac** [comment]
You can install exegol system-wise but exegol rely on docker which is (by default) a single root tenant. So every user of the workstation having the permission to reach docker can interact with every container.
There is also a docker "rootless" setup but itsn't supported officialy by exegol (for now).
Regarding workspace volume, you can managed them as you what but root will always have access everywhere.

**Comment by ghecko** [comment]
Yes, I understand that. However, as I understand it, when I call Exegol, it will always use the user-specific configuration for Exegol (located in their home folder), correct? Is there a way to configure it so that users share the same configuration for Exegol? For example in `/etc/exegol/config`
My main goal is to provide the possibility for users to share workspaces.

**Comment by Dramelac** [comment]
If every user use exegol through `sudo` (instead of being member of `docker` group), everybody will use /root/.exegol/config.yml.
